CRM/Case/Form/Activity.php

```diff
@@ -601,8 +601,16 @@ public function postProcess($params = NULL) {
             $mailStatus = ts("A copy of the activity has also been sent to selected contacts(s).");
           }
           else {
-            $this->_relatedContacts = CRM_Activity_BAO_ActivityAssignment::getAssigneeNames([$vval['actId']], TRUE, FALSE);
-            $mailStatus .= ' ' . ts("A copy of the activity has also been sent to assignee contacts(s).");
+            $actTypeId = civicrm_api3('Activity', 'getvalue', [
+              'return' => 'activity_type_id',
```

Comment:
The activity_type_id is already in both `$params['activity_type_id']` and `$this->_activityTypeId`, so don't need to retrieve it again.

Reply:
Thanks @demeritcowboy , Have updated the changes